src/media/media.service.spec.ts

```diff
@@ -228,4 +228,29 @@ describe('MediaService', () => {
       }
     });
   });
+
+  describe('listUploadsByUser', () => {
+    it('works with one upload from user', async () => {
```

Comment:
One could move both `it`s into a `describe` with the title `works`

Reply:
IMHO that only makes sense if you also have a describe block and tests for "works not".

Reply:
I changed it.